xarray/core/common.py

```diff
@@ -398,11 +398,15 @@ def assign_coords(self, coords=None, **coords_kwargs):
 
         Parameters
         ----------
-        coords : dict, optional
-            A dict with keys which are variables names. If the values are
-            callable, they are computed on this object and assigned to new
-            coordinate variables. If the values are not callable,
-            (e.g. a ``DataArray``, scalar, or array), they are simply assigned.
+        coords : (dict, tuple, optional)
+            If a dict is provided, the keys are the names of the coordinates
+            with the new values to assign. If the values are callable, they are
```

Comment:
```suggestion
        coords : dict, optional
            A dict where the keys are the names of the coordinates
            with the new values to assign. If the values are callable, they are
```
I think you confused this parameter with the `**coords_kwargs` below: the usage with `coords` is
```python
da.assign_coords({"lon_2": ('lon', lon_2)})
```
whereas the `coords_kwargs` form is:
```python
da.assign_coords(lon_2=('lon', lon_2))
```